src/core/api/robot.go

```diff
@@ -108,6 +108,7 @@ func (r *RobotAPI) Post() {
 		return
 	}
 	robotReq.Visible = true
+	robotReq.ByPassPolicyCheck = false
```

Comment:
If `ByPassPolicyCheck` not needed in robotReq, here is not needed too.

Reply:
removed.